lib/auth/autoupdate/autoupdatev1/service.go

```diff
@@ -756,6 +804,30 @@ func (s *Service) RollbackAutoUpdateAgentGroup(ctx context.Context, req *autoupd
 		return nil, trace.Wrap(err)
 	}
 
+	defer func() {
+		var errMsg string
+		if err != nil {
```

Comment:
Need to name the return param for this to work, otherwise you're only capturing the first error from `Authorize`